Part2-API-Schemas/openapi.yaml

```diff
@@ -685,7 +685,7 @@ components:
           minItems: 1
           items:
             type: string
-            enum:
+            example:
```

Comment:
is the following list of profiles a correct example of an array of strings?

Reply:
Well, the array example is under ServiceDescription, this example here lists the currently defined profile strings that can be used in an array. So, I would let the commit pass.